core/src/test/scala/com/nvidia/spark/rapids/tool/qualification/QualificationSuite.scala

```diff
@@ -1030,8 +1030,8 @@ class QualificationSuite extends FunSuite with BeforeAndAfterEach with Logging {
     assert(headersDetailed.size == QualOutputWriter
       .getDetailedHeaderStringsAndSizes(Seq(qualApp.aggregateStats.get), false).keys.size)
     //UnsupportedeExecs and UnsupportedExprs is not present in the file
-    assert(headersDetailed.size -2  == csvDetailedFields.size)
-    assert(valuesDetailed.size - 1 == csvDetailedFields.size) // UnsupportedExprs is empty
+    assert(headersDetailed.size - 3 == csvDetailedFields.size)
+    assert(valuesDetailed.size - 3 == csvDetailedFields.size) // UnsupportedExprs is empty
```

Comment:
We need a comment here to explain why this index got shifted by 2 while everything else was shifted by 1.

Reply:
After working on this for a bit, found that `csvDetailedFields` does not match the expected output files nor the actual output. Opening up a bug